tcms/testruns/tests/test_views.py

```diff
@@ -565,3 +565,157 @@ def test_show_add_cases_to_run(self):
             ]
             for html in html_pieces:
                 self.assertContains(response, html, html=True)
+
+
+class TestCaserunEditMenus(BaseCaseRun):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+    def setUp(self):
```

Comment:
`setUp` is executed before every test method, while `setUpTestData` is executed only once. Given that your testing data doesn't change between the various test methods you probably want this executed only once.